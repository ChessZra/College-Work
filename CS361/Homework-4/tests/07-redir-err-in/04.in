echo This is CS361 >4.out
/bin/echo This also is CS361 >44.out
sed -i s/361/361-Fall-2024/g 4.out 44.out
cat 4.out
cat 44.out
rm 4.out 44.out
file 4.out
file 44.out
