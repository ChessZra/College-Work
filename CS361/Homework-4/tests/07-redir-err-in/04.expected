cs361sh> cs361sh> cs361sh> cs361sh> This is CS361-Fall-2024
cs361sh> This also is CS361-Fall-2024
cs361sh> cs361sh> 4.out: cannot open `4.out' (No such file or directory)
cs361sh> 44.out: cannot open `44.out' (No such file or directory)
cs361sh> 
